conf/development.ini

```diff
@@ -10,6 +10,8 @@ pyramid.default_locale_name = en
 pyramid.includes =
     pyramid_debugtoolbar
 
+exclog.extra_info = true
```

Comment:
This causes `pyramid_exclog` to print a bunch of useful info about the request (headers, params, ...) to the terminal before any exception that it prints, in dev.